client/src/js/components/bhCashBoxFinder.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * The cash box finder component
+ *
+ * @module components/bhCashBoxFinder
+ *
+ * @description a component to deal with cash box, it let you choose a cash box
+ * 
+ **/
+ 
+ //A cash box finder component controller
+ 
+ function cashBoxFinderController (CashboxService, $translate){
+ 	
+ 	var ctrl = this;
+ 	ctrl.types = [
```

Comment:
In this case, it is probably better to simply have the label as the untranslated key and then translate it in the HTML.  That way, your test cases will be more predictable.